contrib/cloud.google.com/go/pubsub.v1/pubsub.go

```diff
@@ -63,15 +64,45 @@ func (r *PublishResult) Get(ctx context.Context) (string, error) {
 	return serverID, err
 }
 
+type receiverConfig struct {
+	// If set, overrides the trace's service name tag.
+	serviceName string
+}
+
+// ReceiveHandlerOption values are passed to WrapReceiveHandler, in order to override tracing defaults.
+type ReceiveHandlerOption interface {
```

Comment:
Please use the same pattern as other integrations do for setting configuration (not an interface).